server/datastore/mysql/vpp.go

```diff
@@ -858,6 +860,17 @@ func (ds *Datastore) UpdateVPPTokenTeams(ctx context.Context, id uint, teams []u
 	})
 
 	if err != nil {
+		mysqlErr := &mysql.MySQLError{}
+		// https://dev.mysql.com/doc/mysql-errors/8.4/en/server-error-reference.html#error_er_dup_entry
+		if errors.As(err, &mysqlErr) && mysqlErr.Number == 1062 {
```

Comment:
Nice! Heads up: we have an `IsDuplicate` function in the mysql/errors.go file that does this check, can probably use that

Reply:
Dang good to know